client/src/components/ModalComponents/TaskModal/TaskModal.tsx

```diff
@@ -16,20 +16,23 @@ import {
   trash,
   scheduleTime,
   timePicker,
+  wrapper,
 } from "./TaskModal.css";
 import "react-datepicker/dist/react-datepicker.css";
 import { FaRegTrashAlt } from "react-icons/fa";
 import { VscChromeClose } from "react-icons/vsc";
 
 import dayjs from "dayjs";
-import { DemoContainer } from "@mui/x-date-pickers/internals/demo";
 import { AdapterDayjs } from "@mui/x-date-pickers/AdapterDayjs";
 import { LocalizationProvider } from "@mui/x-date-pickers/LocalizationProvider";
-import { MobileTimePicker } from "@mui/x-date-pickers/MobileTimePicker";
-import { wrapper } from "../RegisterModal/RegisterModal.css";
+import { TimePicker } from "@mui/x-date-pickers/TimePicker";
+import { toggleTaskModal, useUserInfoStore } from "../../../store/store";
 
 const TaskModal = () => {
-  const [startDate, setStartDate] = useState(new Date());
+  const selectedDate = useUserInfoStore((state) => state.selectedDate);
+  const date = selectedDate ? selectedDate : new Date();
+  const [startDate, setStartDate] = useState(date);
```

Comment:
이렇게 표현해줘도 괜찮을 것 같네요!
```suggestion
  const selectedDate = useUserInfoStore((state) => state.selectedDate);
  const [startDate, setStartDate] = useState(selectedDate ?? new Date());
```